java/com/google/cloud/deploymentmanager/autogen/soy/README.md

```diff
@@ -14,7 +14,7 @@ constructs.
 ## Preprocessing
 
 To support (1) and (2) above, the soy templates are preprocessed by
-[`Preprocessor.java`](http://cs/piper///depot/google3/third_party/java_src/deployment_manager_autogen/java/com/google/cloud/deploymentmanager/autogen/soy/Preprocessor.java).
+[`Preprocessor.java`](http://cs/piper///depot/google3/third_party/java/marketplace_autogen/java/com/google/cloud/deploymentmanager/autogen/soy/Preprocessor.java).
```

Comment:
Not sure if we should replace this with a relative or external link? Can also do in a separate PR.

Reply:
Yeah I'm incrementally trying to remove leakage. (This existed before :) )